change/office-ui-fabric-react-2020-11-06-13-57-58-7.0_picker_aria.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

Comment:
Let's keep it minor since it's a DOM structure addition

Reply:
I wouldn't generally say adding an attribute qualifies as a structure change (that's more about adding/removing/moving elements). But if you'd still rather keep it as a minor change that's fine.

Reply:
Sorry, yes you are correct on the terminology. It's not so much DOM structure as a general DOM change. 

I guess the way I see it is that since aspire to treat DOM attribute removal as a major, we should treat DOM attribute addition as a minor. It's really a stretch, but also it seems better to be conservative on patch vs. minor.

Good to discuss these points though.